source/includes/fundamentals/code-snippets/bson-gradle-versioned.rst

```diff
@@ -0,0 +1,6 @@
+.. code-block:: groovy
+
+   dependencies {
+      compile 'org.mongodb:bson:4.1.0-beta2'
```

Comment:
Why not just 4.1.0?

Reply:
Sorry, will update. I was working from the current reference docs (https://mongodb.github.io/mongo-java-driver/4.1/bson/installation-guide/)
These pages, which have names postfixed with "versioned", will all receive updates when we are ready (close to completion) to peg the new documentation to a specific version of the driver.

Reply:
Updated with 4.1.0